storage/bucket/main.tf

```diff
@@ -1,35 +1,60 @@
-resource "google_storage_bucket" "bucket" {
-  name                        = "${var.project}-${var.name}"
-  project                     = var.project
-  location                    = var.location
-  force_destroy               = true
-  uniform_bucket_level_access = true
-  public_access_prevention    = "enforced"
-  labels                      = var.labels
+resource "google_storage_bucket" "buckets" {
+  for_each = { for k, v in var.buckets : k => v if v.name != "" }
```

Comment:
Module should only create one bucket unless there is a good reason to create multiple buckets at the same time.